packages/build-ui/src/components/input.rs

```diff
@@ -94,3 +95,31 @@ pub fn TextArea(
         }
     }
 }
+
+#[component]
+pub fn BottomBorderInput(
+    label: String,
+    #[props(default = "".to_string())] placeholder: String,
+    value: String,
+    on_change: EventHandler<String>,
+    #[props(default = false)] disabled: bool,
+    #[props(default = false)] invalid: bool,
+    #[props(default = "".to_string())] invalid_message: String,
+    #[props(extends = GlobalAttributes)] attributes: Vec<Attribute>,
+) -> Element {
+    rsx! {
+        div { class: "flex gap-2 w-full items-center", ..attributes,
+            input {
+                "aria-invalid": invalid,
+                class: "text-[17px]/[27px] font-semibold flex-1 border-b border-neutral-80 px-4 py-3 outline-none text-white hover:border-primary focus:border-primary aria-invalid:border-pink placeholder-neutral-80 placeholder-font-xxl disabled:!border-neutral-80",
+                placeholder,
+                value,
+                disabled,
+                oninput: move |e| on_change(e.value().clone()),
+            }
+            if invalid {
+                span { class: "text-[15px]/[23px] text-pink", {invalid_message} }
+            }
```

Comment:
```suggestion
        div { class: "group flex gap-2 w-full items-center", ..attributes,
            "aria-invalid": invalid,
            input {
                class: "text-[17px]/[27px] font-semibold flex-1 border-b border-neutral-80 px-4 py-3 outline-none text-white hover:border-primary focus:border-primary group-aria-invalid:border-pink placeholder-neutral-80 placeholder-font-xxl disabled:!border-neutral-80",
                placeholder,
                value,
                disabled,
                oninput: move |e| on_change(e.value().clone()),
            }
            span { class: "text-[15px]/[23px] text-pink hidden group-aria-invalid:block", {invalid_message} }
```

This `group-aria-*`  may be useful for consistent visualization.